### PR TITLE
Improves Group Description Page

### LINKED ISSUE #1033: Group Page can look better [new UX]
Issues that I found on `/groups/:id`

- Button for removing a user from a group needs some more margin on long usernames
- `Add Member` and `Add new Assignment` should be in same line as their headers, for now, there is wired empty space between header and content of the section
- Margin above group name is bigger than the margin below `Back` button

**Additional context**
image

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by Nitish145**
Fixes #1033 

#### Describe the changes you have made in this PR -
-  Mobile responsive using bootstrap flex utils.
-  grid system for member card responsiveness.
-  Proper Indentation.

### Screenshots of the changes (If any) -
Screenshot from 2020-03-08 00-40-50
Screenshot from 2020-03-08 00-41-12

### CODE REVIEW

File: app/views/groups/show.html.erb [hunk 1]
```diff
@@ -43,26 +44,26 @@
...
+                      <% end %>
+                    </ul>
                   </div>
+                <% end %>
+                <div class="field">
+                  <%= form.hidden_field :group_id, id: :group_member_group_id %>
+                </div>
+                <div class="form-group">
+                  <label for="Email Ids">Email IDs:</label>
+                  <textarea class="form-control" rows="5" id="emails" name="group_member[emails]"></textarea>
```
[the remark this answers is not in the document]
Comment (Nitish145) [reply app/views/groups/show.html.erb]: @nitin10s what should be done here, i haven't changed anything here..

**Comment by Nitish145** [comment]
@nitin10s @aayush-05 reviews please :)

File: app/views/groups/show.html.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitin10s) [reply app/views/groups/show.html.erb]: the "label for" [currently which is "Email Ids" ] should match with the input id [currently which is "emails"]

[the remark this answers is not in the document]
Comment (Nitish145) [reply app/views/groups/show.html.erb]: @nitin10s done !!

**Comment by Nitish145** [comment]
Will do shortly

**Comment by Nitish145** [comment]
@MrBartusek Checkout the changes and let me know if more are required..

**Comment by Nitish145** [comment]
@nitin10s please review :)

**Comment by aayush-05** [comment]
These changes look good, although add a few more fixes too
-See why the font isn't consistent on this page
-Remove the div border (see prototype)
-Decrease the height of the member cards (or even increase the font size for the member name)
-The font on these buttons can also be made a little bigger
image

Also, send the screenshots for the assignment section as well (both pc and mobile).

**Comment by Nitish145** [comment]
Will commit shortly 🙂

**Comment by Nitish145** [comment]
Screenshot from 2020-03-09 18-14-40
Screenshot from 2020-03-09 18-14-00
Screenshot from 2020-03-09 18-13-38
Screenshot from 2020-03-09 18-13-35

**Comment by Nitish145** [comment]
@nitin10s @aayush05 reviews please :)

**Comment by Nitish145** [comment]
@nitin10s what needs to be added/changed here?

**Comment by aayush-05** [comment]
Can you please add how this page looks on mobile display sizes - Iphone 5/SE (you can use chrome simulator)

**Comment by Nitish145** [comment]
There were some noticeable bugs in iphone5/SE display..(Before and after ss are attatched)

## BEFORE
Screenshot from 2020-03-12 20-17-35

## AFTER
Screenshot from 2020-03-12 20-25-21

**Comment by Nitish145** [comment]
@nitin10s @aayush-05 reviews please :)

**Comment by Nitish145** [comment]
Changes corresponding to previous commit:
# BEFORE
Screenshot from 2020-03-12 20-25-21

# AFTER
Screenshot from 2020-03-12 20-54-32

**Pull request merged**